lib/ecto/adapters/postgres/connection.ex

```diff
@@ -682,6 +682,13 @@ if Code.ensure_loaded?(Postgrex) do
       "ALTER TABLE #{quote_table(constraint.prefix, constraint.table)} DROP CONSTRAINT #{quote_name(constraint.name)}"
     end
 
+    def execute_ddl({:create, %View{} = view}) do
+      {query, params, _key} = Ecto.Query.Planner.prepare(view.query, :all, Ecto.Adapters.Postgres, 0)
```

Comment:
I see. I think the planner API is not public, so we cannot access it from adapters. We will need to call those functions at some point before from Ecto, probably in Ecto.Migration, since it would give the best stacktrace in case something goes wrong.